src/coreclr/jit/emit.cpp

```diff
@@ -7986,6 +8073,12 @@ void emitter::emitInitIG(insGroup* ig)
 #if FEATURE_LOOP_ALIGN
     ig->igLoopBackEdge = nullptr;
 #endif
+
+#ifdef DEBUG
+    ig->lastGeneratedBlock = nullptr;
+    // Explicitly call the constructor, since IGs don't actually have a constructor.
+    ig->igBlocks.jitstd::list<BasicBlock*>::list(emitComp->getAllocator(CMK_LoopOpt));
```

Comment:
GCC leg was failing when PR got merged. it looks to be expecting type rather than ctor (function):
```suggestion
    ig->igBlocks.jitstd::list<BasicBlock*>(emitComp->getAllocator(CMK_LoopOpt));
```